buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec

```diff
@@ -56,7 +56,7 @@ Requires: python%{python3_pkgversion}-setuptools
 Requires: python%{python3_pkgversion}-sphinx
 Requires: python%{python3_pkgversion}-sphinx-bootstrap-theme
 Requires: python%{python3_pkgversion}-toml
-Requires: python%{python3_pkgversion}-PyYAML
+Requires: python%{python3_pkgversion}-PyCifRW
```

Comment:
May I ask why we're not just adding PyCifRW but swapping the yaml parser?

If we are making a change here then `Version:        2.1` at the top of the file needs bumping too. Thanks.

Reply:
There is another PyYAML a few lines above.  So I switch the duplicated one to PyCifRW.  Of course, if there is any particular reason that PyYAML need to be installed twice, I will add it back where it was before.

Reply:
Ah yes the diff was collapsed so I didn't spot it. You are quite right here.